sonar-project.properties

```diff
@@ -6,6 +6,6 @@ sonar.sources=src,views
 
 sonar.projectName=limited-partnerships-web
 sonar.exclusions=**/bin/**, **/dist/**, **/coverage/**, **/node_modules/**, **/src/model/**
-sonar.tests=test
-sonar.test.inclusions=**/test/**.spec.*
+sonar.tests=src/test
+sonar.test.inclusions=**/src/test/**.spec.*
```

Comment:
i don't think you need src added to the inclusions as ** should be any path.
i would also add ", **/test/**.test.*" as well as some tests will be called something.test.ts

Reply:
there should be a double asterix befor the bold text